Source/Core/InputCommon/ControllerInterface/SDL/SDL.cpp

```diff
@@ -18,6 +18,14 @@ namespace ciface
 namespace SDL
 {
 
+namespace
+{
+// 10ms = 100Hz which homebrew docs very roughly imply is within WiiMote normal
+// range, used for periodic haptic effects though often ignored by devices
+const u16 RUMBLE_PERIOD = 10;
+const u16 RUMBLE_LENGTH_MAX = 500; // ms: enough to span multiple frames at low FPS, but still finite
```

Comment:
That's fine, I'll change that - I don't care much myself, I was just copying what I saw in another random seemingly-well-written module. :)